docker-compose.yaml

```diff
@@ -1,19 +1,10 @@
 services:
-  wes-data-sharing-service:
```

Comment:
We don't need to build the `wes-data-sharing-service` anymore?

Reply:
Not as part of the test services anymore. `make test` now builds the image and runs the test.py in a container, so the build _does_ still get tested.

Reply:
ohhh ! okay cool